docs/index.md

```diff
@@ -38,6 +38,12 @@ wget -q -O- https://github.com/hugsy/gef/raw/main/scripts/gef-extras.sh | sh
 The script will download (via git) GEF-Extras, and set up your `~/.gef.rc` file so that you can
 start straight away.
 
+If you receive an error mentioning --break-system-packages, you can bypass it by running this command at your own risk:
+
+```bash
+wget -q -O- https://github.com/hugsy/gef/raw/main/scripts/gef-extras.sh | sed 's/\(pip install\)/\1 --break-system-packages/' | sh
```

Comment:
I'm not a huge fan of telling users to use this flag, especially for people who don't really understand what they do.
Maybe trying to use pipx if it exists could be a better solution? Or tell the user to use a virtual environment and provide a way to install the packages in the provided environment? (And maybe put the path of the venv somewhere so that we can load it dynamically at runtime directly in gef-extras to avoid issues)

Reply:
With this PR, just I'm trying to prevent Github issues or queries on Discord regarding this error on install. 

If we want to improve the installation script, it's better to do it in a separate PR, right?

Reply:
I think it's to have issues on discord about how to install gef-extras cleanly for users having this error, instead of telling users (who may have 0 knowledge about pip) to pass this flag to pip.
If you think rewriting this part of the installation script should be done in another PR, I can try to do it later today if you don't want to do it

Reply:
@hugsy opinion?

Reply:
I tend to agree with @ValekoZ on this: I hate how packagers are fighting each other on various distro, but recommending `--break-system-packages` is a bit extreme for me - even though I understand the logic behind.

> Right now, if a user runs the script we have documented, it won't work, and they'll come to complain, haha.

This is only true if people did stuff like `apt install python3-pip` (which is not what they should be doing but whatever). So instead so we should just put a disclaimer stating that gef-extras requires the official `pip` install (not OS packaged). Since there is no [no proper official way](https://packaging.python.org/en/latest/specifications/externally-managed-environments/#externally-managed-environments) I don't think it's our place to recommand something that may break more on their machine along the way

Reply:
oka! closing PR... thx @hugsy  & @ValekoZ